Extension/src/Debugger/attachQuickPick.ts

```diff
@@ -0,0 +1,64 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as util from '../common';
+import * as vscode from 'vscode';
+
+class RefreshButton implements vscode.QuickInputButton {
+    get iconPath(): vscode.Uri {
+        const refreshImagePath: string = util.getExtensionFilePath("assets/Refresh_inverse.svg");
```

Comment:
It seems like this won't appear correctly if the user has a non-dark theme. Do you know how to check if the non-inverse icon should be used?

Reply:
iconPath can return `{ light: Uri; dark: Uri }`

Reply:
Does not seem to be an issue in Light+ theme. 
![image](https://user-images.githubusercontent.com/3953714/49765777-54027880-fc88-11e8-8131-376764252806.png)

Reply:
I just pushed a change that uses different icons for dark and light themes.